pkg/cloudprovider/vsphere/nodemanager_test.go

```diff
@@ -34,10 +36,20 @@ import (
 	cm "k8s.io/cloud-provider-vsphere/pkg/common/connectionmanager"
 )
 
+func (s *SearchIndex) FindByUUID(req *types.FindByUuid) soap.HasFault {
```

Comment:
vcsim already implements `FindByUUID`, you shouldn't need to override as we did for `FindByIp` a while back.

Reply:
Sorry, I was debugging last night trying to figure out that intermittent go test failure and had to step away as it getting late.